code/processes/metrics.q

```diff
@@ -45,7 +45,7 @@ metrics:{[syms]
 
 / check for TP connection
 notpconnected:{[]
-	0 = count select from .sub.SUBSCRIPTIONS where proctype in ((),`tickerplant), active}
+	0 = count select from .sub.SUBSCRIPTIONS where proctype in ((),`segmentedtickerplant), active}
 
```

Comment:
Does this metrics process definitely handle the news messages that arrive at endofperiod / endofday?

Reply:
These functions have now been added